taskiq_sqs/broker.py

```diff
@@ -32,6 +35,12 @@ def __init__(
         self._sqs = boto3.resource("sqs")
         self._sqs_queue: Optional[Queue] = None
 
+        if max_number_of_messages > 10:
```

Comment:
why limit 10?

Reply:
It's [a limitation imposed by AWS](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_ReceiveMessage.html#SQS-ReceiveMessage-request-MaxNumberOfMessages).  Just wanted to catch it early for the user.

Reply:
I see, lgtm outside of this